client/src/components/shop-details/AddDrink.jsx

```diff
@@ -42,6 +44,29 @@ function AddDrink(props) {
     setPriceValue('');
   };
 
+  const addDrinkItem = () => {
+    let obj = {
+      place_id: 'ChIJr0p1HSe5QIYRJbI_fFPj6e0',
+      drink_name: 'Late',
+      rating: 1,
```

Comment:
Server-side I used a variable called "rating" that's a 1 or a 0 to set the rating within the database, but the parameter that the route is expecting from this request is "recommend" which should be a boolean value.